api/integrations/github/client.py

```diff
@@ -12,3 +12,12 @@ def generate_token(installation_id: str, app_id: int) -> str:
     Github(auth=auth)
     token = auth.token
     return token
+
+
+def generate_jwt_token(app_id: int) -> str:  # pragma: no cover
+    github_auth: Auth.AppAuth = Auth.AppAuth(
+        app_id=int(app_id),
```

Comment:
According to function typing annotations, `app_id` is already `int`.

Reply:
Right, since we are not yet using MyPy or another runtime type checker I would argue that, but in this case, it is guaranteed it will be an int so I am removing the explicit casting.